hw/ip/gpio/data/gpio.hjson

```diff
@@ -390,7 +391,14 @@
       ],
     },
     { name: "HW_STRAPS_DATA_IN",
-      desc: "GPIO Input data sampled as straps during cold boot read value",
+      desc: '''
+        GPIO input data that was sampled as straps as the block came out of reset.
+
+        This register depends on the GpioAsHwStrapsEn parameter.
```

Comment:
I guess the register exists either way, so maybe "The behaviour of this register depends ..." ?